app/client/src/workers/Actions.ts

```diff
@@ -257,6 +257,16 @@ const DATA_TREE_FUNCTIONS: Record<
         };
       },
   },
+  postMessageToTargetWindow: function(message: any, targetOrigin: string) {
```

Comment:
Instead of type as `any` for message argument. Can we use `unknown` here? Just as a good practise. https://dmitripavlutin.com/typescript-unknown-vs-any/ 
```suggestion
  postMessageToTargetWindow: function(message: unknown, targetOrigin: string) {
```

Reply:
great suggestion, done